source/create_tgan_model.py

```diff
@@ -0,0 +1,14 @@
+#This script
+
+import pandas as pd
+import numpy as np
+from tgan.model import TGANModel
+
+data = pd.read_csv('../data/census-train.csv',low_memory=False)
+continuous_columns = [0, 5, 16, 17, 18, 29, 38]
+
+tgan = TGANModel(continuous_columns)
+tgan.fit(data)
+
+model_path = '../demos/Model1'
```

Comment:
You could make the data and model_path arguments command line based, but this is not a priority